src/openrct2-ui/windows/TitleMenu.cpp

```diff
@@ -202,6 +203,9 @@ static void window_title_menu_dropdown(rct_window* w, rct_widgetindex widgetInde
             case 3:
                 Editor::LoadTrackManager();
                 break;
+            case 4:
+                context_open_custom_user_content_folder();
```

Comment:
For consistency, please also rename the functions accordingly (you already renamed the ones in the previous PR at my request).